Makefile

```diff
@@ -62,6 +62,10 @@ endif
 endif
 endif
 
+# bin file definitions
+PHPUNIT=php -d zend.enable_gc=0 "$(CURDIR)/../../lib/composer/bin/phpunit"
```

Comment:
In other repos we put ``PWD`` (and in the next line)
Why is this a bit different?